src/clm/commands/train_discriminator.py

```diff
@@ -0,0 +1,141 @@
+"""
```

Comment:
If at this point you have all the information in the output file to generate the ROC curve as well as the confusion matrix, you might as well incorporate it as part of this "command". We won't be testing anything other than the csv files of course, but this will prevent rapid proliferation of these commands as well as further commands/scripts to generate pngs etc.

Reply:
I was thinking of adding all the scripts relating figures inside `scripts/figures`  and adding a `--plot` option to the existing commands to generate graph.